io.github.sockeye_d.godl.yml

```diff
@@ -0,0 +1,51 @@
+id: io.github.sockeye_d.godl
+runtime: org.kde.Platform
+runtime-version: '6.9'
+sdk: org.kde.Sdk
+command: godl
+finish_args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=host
```

Comment:
why does a "godot version manager" need host? i didn't understand what you meant by "it creates, reads, and modifies Godot projects from the filesystem" - then it is not just a version manager?